sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -35,8 +35,6 @@ class ServiceBusClient(object):
      implements a particular interface for getting tokens. It accepts
      :class:`ServiceBusSharedKeyCredential<azure.servicebus.ServiceBusSharedKeyCredential>`, or credential objects
      generated by the azure-identity library and objects that implement the `get_token(self, *scopes)` method.
-    :keyword str entity_name: Optional entity name, this can be the name of Queue or Topic.
```

Comment:
did we remove this? curious because i could see a self.entity_name in the code below.